### PR TITLE
Add command for viewing labels for orgs

### PR DESCRIPTION
**Pull request by sannidhi**
## Description of the Change

[finishes #165425041](https://www.pivotaltracker.com/story/show/165425041)

### CODE REVIEW

**Comment by sannidhi** [comment]
Thank you for the feedback on the PR @acrmp.

Updated changes here: https://github.com/cloudfoundry/cli/pull/1630/commits/94f5a1908b1e227e7b386f0edad013bd55bfc7d9

File: command/v7/labels_command_test.go [hunk 1]
```diff
@@ -46,117 +46,215 @@ var _ = Describe("labels command", func() {
...
 			})
-		})
 
-		When("there is an error retrieving the application", func() {
-			BeforeEach(func() {
-				fakeActor.GetApplicationByNameAndSpaceReturns(
-					v7action.Application{},
-					v7action.Warnings([]string{"some-warning-1", "some-warning-2"}),
-					errors.New("boom"))
+			It("checks that the user is logged in and targeted to an org and space", func() {
```
Comment (acrmp) [inline command/v7/labels_command_test.go]: The example wording is wrong - this is only checking that the user is logged in, not that they are targeted to an org and space.

File: command/v7/labels_command.go [hunk 2]
```diff
@@ -34,16 +42,39 @@ func (cmd *LabelsCommand) Setup(config command.Config, ui command.UI) error {
...
-	username, err := cmd.Config.CurrentUserName()
+	}
+	cmd.UI.DisplayWarnings(warnings)
 	if err != nil {
 		return err
 	}
 
+	cmd.printLabels(labels)
+	return nil
+
```
Comment (bwasmith) [inline command/v7/labels_command.go]: Can you please remove this whitespace?

**Comment by bwasmith** [comment]
Thanks for your PR! After these changes are addressed, @acrmp and I think this should be good to merge.

File: command/v7/v7fakes/fake_get_label_actor.go [hunk 3]
```diff
@@ -0,0 +1,207 @@
+// Code generated by counterfeiter. DO NOT EDIT.
```
[the remark this answers is not in the document]
Comment (sannidhi) [reply command/v7/v7fakes/fake_get_label_actor.go]: We deleted the fake_label_actor and used the fake_get_label_actor instead to match existing pattern in code.

**Comment by sannidhi** [comment]
@bwasmith - We updated the PR addressing the comments, please let us know when you merge this in, thank you!

File: command/v7/labels_command.go [hunk 4]
```diff
@@ -10,15 +10,23 @@ import (
 	"code.cloudfoundry.org/cli/command/v7/shared"
 	"code.cloudfoundry.org/cli/types"
 	"code.cloudfoundry.org/cli/util/ui"
+	"fmt"
 )
 
+//go:generate counterfeiter . GetLabelActor
+
+type GetLabelActor interface {
```
Comment (bwasmith) [inline command/v7/labels_command.go]: The convention we are trying to follow is that the Actor for `FooCommand` is `FooActor`. We plan to merge this and make the change ourselves to save the additional back-and-forth. 

See [`command/v7/domains.go`](https://github.com/cloudfoundry/cli/blob/92ecc3a56b6971696d84ad908f204b62ed8b765a/command/v7/domains_command.go) for an example of this.

@sannidhi , you  explained that you and @ericpromislow  tried to fix the fakes by adding this "GetLabelActor" last week in Seattle. Sorry I couldn't have just given you this feedback then. Was there another convention that you had seen elsewhere?

**Comment by bwasmith** [comment]
Also, these changes were merged but were missing an import.

This unfortunately missed our Travis CI unit tests because our tests are not currently set up to run V7. In the future, we would like to have the automated test suite/PR builder in better shape. But in the meantime, we would recommend running `make units` to make sure that things compile happily.

**Comment by sannidhi** [comment]
Hmmm, we ran make units before pushing, wonder how this was not caught. Is
there anything we need to do now?

File: command/v7/labels_command.go (hunk 4 above)
Comment (sannidhi) [reply command/v7/labels_command.go]: @bwasmith  - Sorry just saw this. We saw there was a SetLabelActor and a DeleteLabelActor so we went with ActionActor (where get was the action in our case).

**Pull request merged**